frontends/ol-search-ui/src/components/SearchInput.tsx

```diff
@@ -33,12 +32,13 @@ interface SearchInputProps {
 }
 
 const searchIconAdjustments = {
-  fontSize:  "150%",
+  fontSize:  "300%",
```

Comment:
I noticed that other values make the search button not full-height. Here's 250%:

<img width="569" alt="Screenshot 2023-03-17 at 1 23 18 PM" src="https://user-images.githubusercontent.com/9010790/225975250-6bbba14a-4dd6-41f5-9dfb-60ba95f5d6fa.png">

If 300 is the value we want, then keep it 300. But we should still decouple the font-size and height of the button, which can be done via
```
const adornmentAdjustment = {
  alignSelf: "stretch",
  maxHeight: "initial",
  height:    "initial"
}
<InputAdornment sx={adornmentAdjustment} />
```

IMO, this styling is OK to keep in the component class: the button taking full height of the input is a feature of the component, not something someone would want to change externally.